packages/fast-components-class-name-contracts-msft/src/managed-classes.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Class names under a given object keys
+ */

```

Comment:
I think you'll want to take a dependency on the base and then just use these interfaces from there - that will prevent duplicating these interfaces.